### PR TITLE
Revert RHSSO 7.4 upgrade. Revert "Merge pull request #760 from davidffrench/INTLY-7020"

### PR DESCRIPTION
**Pull request by davidffrench**
This reverts merge commit 6c6ec8dada5ec165ad54a14c28ad389d4a6020c3, reversing
commits ff1de97bcd70dee4e7971df2fb96df22270ad270 and d1f7c9ae1e325ac123ae07311f99fd195d9affe6.

# Description
**JIRA:** https://issues.redhat.com/browse/INTLY-8170

Due to issues found with the update of RHSSO 7.4 in RHMI, the decision has been made to roll back to RHSSO 7.3 to give sufficient time to investigate and resolve the outstanding issues.

## Type of change
- [x] Bug fix (non-breaking change which fixes an issue)

## Checklist
- [ ] This change requires a documentation update 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added a test case that will be used to verify my changes 
- [ ] Verified independently on a cluster by reviewer

### CODE REVIEW

File: pkg/apis/integreatly/v1alpha1/rhmi_types.go [hunk 1]
```diff
@@ -85,8 +85,8 @@ var (
 	OperatorVersionAMQOnline           OperatorVersion = "1.4"
 	OperatorVersionMonitoring          OperatorVersion = "1.1.5"
 	OperatorVersionSolutionExplorer    OperatorVersion = "0.0.55"
-	OperatorVersionRHSSO               OperatorVersion = "10.0.0"
-	OperatorVersionRHSSOUser           OperatorVersion = "10.0.0"
+	OperatorVersionRHSSO               OperatorVersion = "9.0.2"
```
Comment (JameelB) [inline pkg/apis/integreatly/v1alpha1/rhmi_types.go:88]: should these be `9.0.3`?

File: go.mod [hunk 2]
```diff
@@ -81,3 +82,6 @@ replace (
 replace github.com/openshift/api => github.com/openshift/api v3.9.1-0.20190904142825-922d36944de3+incompatible
 
 replace github.com/docker/docker => github.com/moby/moby v0.7.3-0.20190826074503-38ab9da00309 // Required by Helm
+
+// Remove when PR https://github.com/keycloak/keycloak-operator/pull/171 is merged
+replace github.com/keycloak/keycloak-operator => github.com/integr8ly/keycloak-operator-rhmi v0.0.0-20200505143708-9f0afbb299cd
```
[the remark this answers is not in the document]
Comment (davidffrench) [reply go.mod:87]: Unfortunately not. This comment should say "Remove when updating to the operator release PR https://github.com/keycloak/keycloak-operator/pull/171 is included in"

File: pkg/apis/integreatly/v1alpha1/rhmi_types.go (hunk 1 above)
Comment (davidffrench) [reply pkg/apis/integreatly/v1alpha1/rhmi_types.go:88]: Good question. `9.0.3` is our own release of the keycloak operator from our fork (https://github.com/integr8ly/keycloak-operator-rhmi). There is no 9.0.3 release from the upstream keycloak-operator (https://github.com/keycloak/keycloak-operator/releases). Either one is fine in my opinion and I leaned towards `9.0.2` initially as this maps to the upstream version.

**Comment by JameelB** [comment]
/lgtm

Verified on OSD: 

**Cluster SSO**
- rhsso 7.3 has been installed successfully
  image
  image
- Verified: Able to login as customer-admin/dev user to openshift and product consoles.

**User SSO**
- user-sso 7.3 has been installed successfully
  image
  image
- Verified: able to login as customer-admin/dev user to the user-sso admin console
   - customer-admin is able to view all realms in user-sso
   - dev user can only view the `master` and the realms it has created.
- Verified: able to create a realm as a customer-admin/dev user.
- Verified: link and product details in solution explorer is correct
   image

**Pull request merged**